components/infobox/commons/infobox_league.lua

```diff
@@ -327,24 +327,25 @@ function League:_createLocation(args)
 	return content
 end
 
-function League:_createSeries(frame, series, abbreviation, isFirst, icon, iconDark)
+function League:_createSeries(series, abbreviation, isFirst, icon, iconDark)
```

Comment:
rename `isFirst` to `setVariable ` or similar.

Reply:
Would've been better to do so in a follow-up PR. I think it's best if these comments are avoided for a PR in general, they are annoying to get since they're not related to your actual changes

Reply:
On that note, `shouldSetVariable` is better.

Reply:
> Would've been better to do so in a follow-up PR. I think it's best if these comments are avoided for a PR in general, they are annoying to get since they're not related to your actual changes

Roger